modules/openid_connect/app/models/openid_connect/provider.rb

```diff
@@ -3,26 +3,28 @@ class Provider < AuthProvider
     include HashBuilder
 
     OIDC_PROVIDERS = %w[google microsoft_entra custom].freeze
-    DISCOVERABLE_ATTRIBUTES_MANDATORY = %i[authorization_endpoint
-                                           userinfo_endpoint
-                                           token_endpoint
-                                           issuer].freeze
-    DISCOVERABLE_ATTRIBUTES_OPTIONAL = %i[end_session_endpoint jwks_uri].freeze
-    DISCOVERABLE_ATTRIBUTES_ALL = DISCOVERABLE_ATTRIBUTES_MANDATORY + DISCOVERABLE_ATTRIBUTES_OPTIONAL
+    DISCOVERABLE_STRING_ATTRIBUTES_MANDATORY = %i[authorization_endpoint
```

Comment:
Throwing a useless opinion here, but I don't feel the change in name here makes it any more meaningful. Do we somehow differentiate `string` from non `string` attributes?

Reply:
Yes we do, this was the reason that I added this naming change. So far these constants were impacting all attributes equally and the constants are used fot things like "create string accessors for all of those".

However, the attribute that I just added was an array of strings, not a string. Thus I did the renaming to indicate "that's not all discoverable attributes, but just discoverable attributes of type string".

Reply:
Fair enough. Sounds correct to me. :clap: